app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Greeting from './Greeting'; 
+
+const App = () => { 
+  return (
+    <>
+    <Routes> 
+      <Route path="/" element={ <Greeting  />} />
+    </Routes>
+    </>
+  );
+}
+
+export default app;
```

Comment:
- You have done great fixing the errors pointed out by the previous reviewers :+1: But your application is still not running properly and displaying greetings. :+1: 

- [ ] Looking at the screenshot below, you can see that I have a white screen after starting your app with an error in the console `Uncaught ReferenceError: app is not defined`. This is because you are exporting `app` while the component name is `App`. the correct code for line 15 should be `export default App;`. 
## I'll suggest you fix this and actually run the app to make sure everything is ruining in your PC and displaying greetings before requesting for review. :+1: :100: 

- - Screenshot reference:
![Selection_023](https://github.com/DianaBeki/hello-rails-react/assets/107632780/b5d9a11f-864d-49c7-941b-3ccd2fae71e8)

Reply:
Fixed.